driver/csiplugin/connectors/rest_v2.go

```diff
@@ -752,6 +850,53 @@ func (s *SpectrumRestV2) CreateS3CacheFileset(ctx context.Context, filesystemNam
 	return nil
 }
 
+func (s *SpectrumRestV2) CreateNodeMappingAFMWithCos(ctx context.Context, exportMapName string, gatewayNodeName string, bucketInfo map[string]string) error {
+	loggerID := utils.GetLoggerId(ctx)
+	klog.V(4).Infof("[%s] rest_v2 CreateNodeMappingAFMWithCos.exportMapName:[%s],gatewayNodeName:[%s], bucketInfo: %v :", loggerID, exportMapName, gatewayNodeName, bucketInfo)
```

Comment:
bucketInfo should never be logged directly, as it contains secret and access keys.

Reply:
done